frontend/webpack.config.ts

```diff
@@ -7,22 +7,35 @@ import * as MiniCssExtractPlugin from 'mini-css-extract-plugin';
 import * as VirtualModulesPlugin from 'webpack-virtual-modules';
 
 import { resolvePluginPackages, getActivePluginsModule } from '@console/plugin-sdk/src/codegen';
+import { Configuration as WebpackDevServerConfiguration } from 'webpack-dev-server';
+
+interface Configuration extends webpack.Configuration {
+  devServer?: WebpackDevServerConfiguration;
+}
+require('dotenv').config(); // load .env variables
 
 const MonacoWebpackPlugin = require('monaco-editor-webpack-plugin');
 
 const NODE_ENV = process.env.NODE_ENV;
+const HOT_RELOAD = process.env.HOT_RELOAD === 'true';
```

Comment:
it is false unless `HOT_RELOAD === 'true'`

Reply:
to avoid this confusion in the future, we can leave:
```
const HOT_RELOAD = process.env.HOT_RELOAD;
```
like in the above line, and do the `HOT_RELOAD === 'true'` when we use it below, WDYT ?

Reply:
updated